wasm2c/wasm-rt-impl.h

```diff
@@ -31,6 +31,14 @@ extern jmp_buf g_jmp_buf;
 /** Saved call stack depth that will be restored in case a trap occurs. */
 extern uint32_t g_saved_call_stack_depth;
 
+#if WASM_RT_MEMCHECK_SIGNAL_HANDLER_POSIX
+#define WASM_RT_SETJMP(buf) sigsetjmp(buf, 1)
+#define WASM_RT_LONGJMP(buf, val) siglongjmp(buf, val)
+#else
+#define WASM_RT_SETJMP(buf) setjmp(buf)
+#define WASM_RT_LONGJMP(buf, val) longjmp(buf, val)
```

Comment:
What was this needed?   It seems like it would be needed if the sigmask was modified between the a try and a catch, but is that possible?   Isn't the sigmask just set once startup and never again?

Reply:
I'm not 100% sure, but using regular setjmp/longjmp didn't work